tox.ini

```diff
@@ -17,7 +17,7 @@ per-file-ignores =
 [testenv:test_integration]
 
 commands =
-    pytest test/integration/test_dropbox.py
+    pytest test/integration/
```

Comment:
I changed this because integration and unit tests are in standalone directories. We can add new tests into separate files